Assignment1_Q2/restOperator.js

```diff
@@ -0,0 +1,11 @@
+var colors = ["Violet", "Indigo", "Blue", "Green", "Yellow"];  
+      
+// destructuring assignment  
+var [a,b,...args] = colors;  
+
+var moreColors = ["Black", "White", ...colors];
+
+console.log(a);   
+console.log(b);   
+console.log(args);  
```

Comment:
Please add outputs using comments for better understanding.

Reply:
Added the outputs using comments for better understanding.